src/Adapter/Module/PaymentModuleListProvider.php

```diff
@@ -39,7 +36,7 @@
 final class PaymentModuleListProvider implements PaymentModuleListProviderInterface
 {
     /**
-     * @var AddonRepositoryInterface
+     * @var ModuleRepositoryInterface
      */
     private $addonRepository;
```

Comment:
Only a suggestion to get rid of `addonRepository` naming:
```suggestion
    private $moduleRepository;
```